### DISCUSSION TITLE
My SSH keys not working

### DISCUSSION THREAD

**Discussion by Ralu1** [community/community#43599] [Other Feature Feedback, Questions, & Ideas]
I'm trying to add a SSH key but as soon as I press enter it say that:key is invalid. You must supply a key in openSSH public key format what do I do plz

**Comment by airtower-luna** [comment]
Most likely you're not copying the right thing, but obviously I can't confirm that without seeing what it is. The text you're pasting should start with something like `ssh-rsa AA...` (`rsa` could be replaced with some other key type, e.g. `ed25519`).

**Comment by icaroharry** [comment]
Hey @Ralu1 :) 

Check if you're copying the whole key. Sometimes, when copying from the terminal, you might miss some of the characters.

Another common mistake is to input the private key instead of the public one. For other possible solutions, you can check [this discussion](https://github.com/orgs/community/discussions/23089).

Also, a tip for you: when asking for help, try to provide as much information as you can. Which OS are you using? Which commands did you try? etc

👋
